### DISCUSSION TITLE
启动immortalwrt时无法自动进入系统

### DISCUSSION THREAD

**Discussion by jerry-tech222** [immortalwrt/immortalwrt#1366] [General]
出现ESC to return previous menu，要按了ESC健才会出现the highlighted entry will be executed automatically in 2S。这时才会倒数2秒进入系统。请教怎样去除ESC to return previous menu这个提示呢？谢谢！版本是23.05.1，最新的23.05.2也一样的问题。

**Comment by lewishuang14** [comment]
我也存在同样的问题。在物理机安装，使用的镜像是23.05.4-x86-64-generic-squashfs-combined-efi.img。

**Reply by jerry-tech222** [reply]
快尝试下面大神提供的方法吧，我一下就成功了！很开心！

**Reply by lewishuang14** [reply]
一试就好起来了。

**Comment by ikangi** [comment]
应该是跟个别bios不兼容导致的串口交互问题

因为启用了 terminal_input，GRUB 一直在等待串口输入，暂停了自动启动。

可以试试临时禁用串口输入

进入系统后编辑 grub.cfg
`vi /boot/grub/grub.cfg`

按i进入编辑模式，删除
`terminal_input console serial;`

保存修改，按ESC退出编辑模式后输入
`:wq`


然后重启试一下

**Reply by NanWu00** [reply]
非常感谢你的回答解决困扰我好几天的问题，我在引导上折腾了很久。

**Reply by jerry-tech222** [reply]
谢谢！我按您提供的方法已经成功搞好了，效果很满意！

**Reply by jerry-tech222** [reply]
出现这个问题的是一台3865u的小主机，我试过n5000和i3 1005g1的主机，都没有这个问题。再次感谢大佬的指导！

**Reply by bling-yshs** [reply]
太感谢你了<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
